typescript/web/src/components/logo/logo.tsx

```diff
@@ -9,22 +9,65 @@ export const Logo = React.forwardRef<
   const { iconColor = "brand", logoOnly, ...rest } = props;
   return (
     <chakra.svg
-      aria-hidden
-      viewBox={logoOnly ? "0 0 84 84" : "0 0 393 84"}
+      viewBox={logoOnly ? "0 0 188 191" : "0 0 897 191"}
```

Comment:
While on one side you've created an `UkranianLogo` component, on the other you've replaced some hardcoded values of the regular Logo component.

In-order to simplify the rollback even if we change some code based on the logo, can you please either:
* keep the official logo as `OfficialLogo` and the new one as `UkranianLogo` (and use the correct one in a `Logo` component)
or
* Don't change the original viewBox dimensions and use an SVG of the exact same dimensions than the official one.

Reply:
@Jordanlelay You've _self-resolved_ this comment but there is only 1 logo SVG in the code (i.e. the official version has been removed) and the viewBox is different than what is was with the original version. Am I missing something?

What will we have to do once we'll have to go back to the official version? Will it be doable in ~5mn?

Reply:
You asked me to merge both logos file, right ? So now there is only: the Ukrainian one.
Is there some problem using a `git revert` with the work I've done ?
Even manually, it should not take more than 10 minutes to go back to the original logo

Reply:
> You asked me to merge both logos file, right ?

Not exactly:

> keep the official logo as OfficialLogo and the new one as UkranianLogo (and use the correct one in a Logo component)

Which means that we'd have 3 components:
* **_OfficialLogo_** - The official logo
* **_UkranianLogo_** - The Ukranian version
* _**Logo**_ - A component which just exposes either OfficialLogo or UkranianLogo component depending on what we'd like to display for now. I've not specified more details to let you free of doing the change as quick as you believe but we could simply switch between one version or another depending on a `NEXT_PUBLIC_APP_LOGO=ukranian` env variable.

Contrary to a git revert, this would prevent possible conflicts and bugs as we don't have any idea about how much time we'll keep the temporary logo and how much the code will evolve until then.

Reply:
Anyway, as the PR is already taking a lot of time and risks of issues are limited, we can stay with the implementation you've done if you feel like it'd take too much time for you to do the changes 🚀